WCS/build.properties

```diff
@@ -1,3 +1,8 @@
 project_name = Sample
 project_base = src
 project_target = target
+
+# All Environment configurations below
+
+DEPLOY_ENV = "DEV"
+DEV1_SERVER = albd_dev1
```

Comment:
QA and UAT server information needed below. Otherwise deploy scripts will not work.